test/index.js

```diff
@@ -9,6 +9,17 @@ const zlib = require('zlib')
 const parseString = require('xml2js').parseString
 const FileSystem = require('../')
 
+test('type and plugin_name are accessible', t => {
+  const s3fs = new FileSystem()
```

Comment:
this is not right. We need to be able to access `Filesystem.type` instead of `s3fs.type` We already have `s3fs.type`